### PR TITLE
Clarify registry content for #3189

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

## What type of PR is this ? 

- [X ] Bug fix
- [ ] Enhancement

## What does this PR do ?
Clarifies image registry content for #3189. 

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/3189

## Does this PR require a documentation change ?
Yes.

## Any special notes for your reviewer ?
Acrolinx checked and run up locally.

### CODE REVIEW

**Comment by micgibso** [comment]
@markcor11 Please review, thx.

File: docs/_documentations/che-setupregistries.md [hunk 1]
```diff
@@ -8,26 +8,37 @@ permalink: che-setupregistries.html
...
+- When you are using Codewind remotely
+- When you are using Che to build Codewind style projects
 
-## Prerequisite: Determining if you need an image registry 
-1\. First, determine whether you need to add an image registry. The following scenarios in Codewind require you to specify which image registry is in use:
-- In the following scenario, configure Codewind with an image registry to push the application image to that specific image registry:
+### Determining whether you need an image registry for Che
+You need an image registry for Che if:
+- In the following scenario, you want to configure Codewind with an image registry to push the application image to that specific image registry:
     - Run Codewind on a remote Kubernetes cluster, including Che, to develop a Codewind style project.
```
Comment (markcor11) [inline docs/_documentations/che-setupregistries.md:27]: This section is talking about Che specifically so could we remove line 26 and just change 27 to say:

When developing Codewind style projects.

File: docs/_documentations/che-setupregistries.md [hunk 2]
```diff
@@ -8,26 +8,37 @@ permalink: che-setupregistries.html
...
-- In the following scenario, configure Codewind to use the credentials for a specific image registry:
+- In the following scenario, you want to configure Codewind to use the credentials for a specific image registry:
     - Run Codewind on a remote Kubernetes cluster, including Che, to develop an Appsody style project.
```
Comment (markcor11) [inline docs/_documentations/che-setupregistries.md:29]: Same as above remove #28 and replace #29 with 

When developing Appsody style projects

File: docs/_documentations/che-setupregistries.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/che-setupregistries.md:27]: Changed to:
You need an image registry for Che if:
- You are developing Codewind style projects.

File: docs/_documentations/che-setupregistries.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/che-setupregistries.md:29]: Removed and replaced, the section now reads:

### Determining whether you need an image registry for Che
You need an image registry for Che if:
- You are developing Codewind style projects.
- When developing Appsody style projects, the image registry for the Appsody stack requires credentials.

**Comment by micgibso** [comment]
@markcor11 Comments addressed on new commit.

**Pull request merged**